framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/Domain/Repositories/EntityFrameworkCore/EfCoreRepository.cs

```diff
@@ -265,5 +343,18 @@ public virtual async Task DeleteAsync(TKey id, bool autoSave = false, Cancellati
 
             await DeleteAsync(entity, autoSave, cancellationToken);
         }
+
+        public async Task DeleteManyAsync([NotNull] IEnumerable<TKey> ids, bool autoSave = false, CancellationToken cancellationToken = default)
+        {
+            foreach (var id in ids)
+            {
+                await DeleteAsync(id, cancellationToken: cancellationToken);
```

Comment:
`DeleteAsync(id...)` queries the entity by id first. Then deletes it using `DeleteAsync(entity...)`. See the implementation above.

So, it would be much performant if we query entities via single query, like `...Where(x => ids.Contains(x.Id).ToListAsync()`, then send all entities to `DeleteManyAsync(entities...)` method.